scripts/runClientBuild.js

```diff
@@ -80,12 +76,6 @@ async.series([
   function(next) {
     createClientBuild(webpackClientNoReactMinConfig, next);
   },
-  function(next) {
-    createClientBuild(webpackClientNoDepsConfig, next);
-  },
-  function(next) {
-    createClientBuild(webpackClientNoDepsMinConfig, next);
-  },
 ], function(error) {
```

Comment:
I would rather see this whole `async.series` replaced with `Promise`:
```
Promise.resolve()
  .then(createClientBuild(webpackClientFullConfig))
    ...
  .catch(() => {
    if (error) {
      console.error('\nAn error occured, unable to create a server build.'.red.bold);
      throw error;
    }

    console.log('\nAll good!\n'.green.bold);
  });
```

Same also goes for `async.series` usage everywhere else.